content/shmem_init_thread.tex

```diff
@@ -0,0 +1,78 @@
+\apisummary{
+Initializes the \openshmem{} library, similar to SHMEM\_INIT(), and in addition
+perform the initialization required for thread-safe invocation of \openshmem{} functions.}
+
+\begin{apidefinition}
+
+\begin{Csynopsis}
+int shmem_init_thread(int requested, int *provided);
+\end{Csynopsis}
+
+\begin{Fsynopsis}  
+INTEGER requested, provided
+     
+SHMEM_INIT_THREAD(requested, provided)
+\end{Fsynopsis}  
+
+\begin{apiarguments}
+\apiargument{IN}{requested}  {The thread level support requested by the user.
+The correct values are SHMEM\_THREAD\_SINGLE, SHMEM\_THREAD\_FUNNELED, or SHMEM\_THREAD\_MULTIPLE}
+\apiargument{OUT}{provided}{The thread level support provided by the \openshmem{} implementation.}
+\end{apiarguments}
+
+
+\apidescription{
+    \FUNC{shmem\_init\_thread}  initializes the \openshmem{} library similar to
+    \FUNC{shmem\_init}, and in addition perform the initialization required for
+    thread-safe invocation of \openshmem{} functions. The argument
+    \textit{requested} is used to specify the desired level of thread support.
+    The function returns the support provided by the library. There are four 
```

Comment:
Are there four or three? You only call out SHMEM_THREAD_{SINGLE, FUNNELED, MULTIPLE} under the API arguments, but also include SERIALIZED below.